### PR TITLE
Wealthbox - New Components

### LINKED ISSUE #6241: [APP] Wealthbox CRM
**Name of app / service**
Wealthbox CRM

**Link to developer documentation**
http://dev.wealthbox.com/#introduction-introduction

**Context**
Wealthbox CRM indexed in Project App Store sheet

**Triggers**
New Contact
New Event
New Opportunity
New Task

**Actions**
Create a new contact
Create a new event 
Create a new task
Create a new opportunity

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #6241 

Triggers

- New Contact Created
- New Event Created
- New Opportunity Created
- New Task Created

Actions

- Create Contact
- Create Event
- Create Task
- Create Opportunity

### CODE REVIEW

File: components/wealthbox/actions/create-event/create-event.mjs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      type: "string",
+      label: "Starts At",
+      description: "A timestamp signifying when the event starts. Example `2015-05-24 10:00 AM -0400`",
+    },
+    endsAt: {
+      type: "string",
+      label: "Ends At",
+      description: "A timestamp signifying when the event ends. Example `2015-05-24 10:00 AM -0400`",
+    },
+    description: {
```
Comment (luancazarine) [inline components/wealthbox/actions/create-event/create-event.mjs:26]: this prop is not in use

File: components/wealthbox/actions/create-opportunity/create-opportunity.mjs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    targetClose: {
+      type: "string",
+      label: "Target Close",
+      description: "A string representing the date/time when the opportunity should close. Example `2015-05-24 10:00 AM -0400`",
+    },
+    probability: {
+      type: "string",
+      label: "Probability",
+      description: "A number representing the chance the opportunity will close, as a percentage",
+    },
```
Comment (luancazarine) [inline components/wealthbox/actions/create-opportunity/create-opportunity.mjs:25]: the type is string but the description says that it is a number, is that right?

File: components/wealthbox/actions/create-task/create-task.mjs [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import wealthbox from "../../wealthbox.app.mjs";
+
+export default {
+  key: "wealthbox-create-task",
+  name: "Create Task",
+  description: "Create a new task. [See the documentation](http://dev.wealthbox.com/#tasks-collection-get)",
```
Comment (luancazarine) [inline components/wealthbox/actions/create-task/create-task.mjs:6]:
```suggestion
  description: "Create a new task. [See the documentation](http://dev.wealthbox.com/#tasks-collection-post)",
```

File: components/wealthbox/sources/common/common.mjs [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      throw new Error("generateMeta is not implemented");
+    },
+  },
+  async run() {
+    const lastCreated = this._getLastCreated() || 0;
+    let maxLastCreated = lastCreated;
+    let total;
+    const params = {
+      per_page: 25,
+      page: 1,
```
Comment (luancazarine) [inline components/wealthbox/sources/common/common.mjs:56]: you aren't incrementing the page, is that right?

**Review by luancazarine — CHANGES_REQUESTED** [review]
HI @michelle0927, thanks for your contribution. I just left some coments.

File: components/wealthbox/sources/common/common.mjs (hunk 4 above)
Comment (michelle0927) [reply components/wealthbox/sources/common/common.mjs:56]: Oops, thanks. :)

File: components/wealthbox/actions/create-opportunity/create-opportunity.mjs (hunk 2 above)
Comment (michelle0927) [reply components/wealthbox/actions/create-opportunity/create-opportunity.mjs:25]: It's a number, but not necessarily an integer, so string works.

**Review by luancazarine — APPROVED** [review]
LGTM!

**Comment by vunguyenhung** [comment]
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[Wealthbox_CRM_6241_1342.pdf](https://github.com/PipedreamHQ/pipedream/files/11514214/Wealthbox_CRM_6241_1342.pdf)

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Wealthbox_CRM_6241_1043.pdf](https://github.com/PipedreamHQ/pipedream/files/11526844/Wealthbox_CRM_6241_1043.pdf)

**Pull request merged**